src/boot/efi/secure-boot.c

```diff
@@ -32,10 +32,46 @@ SecureBootMode secure_boot_mode(void) {
         return decode_secure_boot_mode(secure, audit, deployed, setup);
 }
 
+/*
+ * Custom mode allows to change secure boot certificate databases db, dbx, KEK and PK without the variable
+ * updates being signed.  When enrolling certificates to an unconfigured system (no PK present yet) writing
+ * db, dbx and KEK updates without signature works fine even in standard mode.  Writing PK updates without
+ * signature requires custom mode in any case.
+ *
+ * Enabling custom mode worky only if a user is physically present.  Note that OVMF has a dummy
```

Comment:
Can you add a link to some spec or document that explains this mode?

Also there are some double spaces in this comment

Reply:
thanks, looks fine then